src/Model/EventManager.php

```diff
@@ -62,4 +61,37 @@ public function selectEventsToCome(): array
 
         return $this->pdo->query($statement)->fetchAll();
     }
+  
+    /**
+     * Insert an Event
+     * @param array $events
+     * @return int
+     */
+    public function insertEvent(array $events): int
+    {
+        // prepared request
+        $statement = $this->pdo->prepare("INSERT INTO $this->table 
```

Comment:
tu es déjà dans EventManager, ici le $this->table est pas vraiment justifié, tu ferais mieux de mettre un self::TABLE

Reply:
Plus compliqué à mettre en place (Fermetrue et concaténation), car étant dans une requete avec les doubles qutoes il l'interprète. donc je me suis permis de le laisser sous cette forme.